server/src/Api/appsettings.Development.json

```diff
@@ -16,5 +16,8 @@
         }
       }
     ]
+  },
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=localhost; Database=studyzen_db; Trusted_Connection=True; MultipleActiveResultSets=True; TrustServerCertificate=True;"
```

Comment:
do we really need `MultipleActiveResultSets=True;`? we aren't really doing any lazy loading so far as I have seen